spec/support/components/work_packages/relations.rb

```diff
@@ -171,7 +171,7 @@ def openChildrenAutocompleter
           find('.wp-inline-create--reference-link', text: I18n.t('js.relation_buttons.add_existing_child')).click
 
           # Security check to be sure that the autocompleter has finished loading
-          page.find '.ng-dropdown-panel-items'
+          sleep 3
```

Comment:
The `retry_block` takes better care of ensuring the items are there. This was caused by the ng-select having basically no width and resulting in non-visible items

Reply:
The test was failing because we do not open the ng-select per default any more. Thus we cannot expect die items to be visible. I added the `sleep` as a heuristic because of the comment, but I guess we can skip it..